src/components/TodoList/TodoList.tsx

```diff
@@ -1,224 +1,107 @@
-/* eslint-disable */
 import React from 'react';
+import { useAppSelector } from '../../app/hooks';
+import classNames from 'classnames';
+import { useDispatch } from 'react-redux';
+import { setCurrentTodo } from '../../features/currentTodo';
+import { Status } from '../../types/Status';
 
 export const TodoList: React.FC = () => {
+  const todos = useAppSelector(state => state.todos);
+  const { status, query } = useAppSelector(state => state.filter);
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const dispatch = useDispatch();
+  const filteredTodos = todos
+    .filter(todo => {
+      switch (status) {
+        case Status.Completed:
+          return todo.completed;
+        case Status.Active:
+          return !todo.completed;
+        default:
+          return todo;
```

Comment:
Returning 'todo' here is incorrect. The default case should return 'true' to include all todos when no specific status filter is applied.